extra/exports

```diff
@@ -450,6 +453,39 @@ title_set_status
 title_update
 tnt_default_cert_dir_paths
 tnt_default_cert_file_paths
+tnt_dt_add_months
+tnt_dt_add_quarters
+tnt_dt_add_years
+tnt_dt_days_in_month
+tnt_dt_days_in_quarter
+tnt_dt_days_in_year
+tnt_dt_dom
+tnt_dt_dow
+tnt_dt_doy
+tnt_dt_from_rdn
+tnt_dt_from_struct_tm
+tnt_dt_from_yd
+tnt_dt_from_ymd
+tnt_dt_from_yqd
+tnt_dt_from_ywd
+tnt_dt_leap_year
+tnt_dt_month
+tnt_dt_parse_iso_date
+tnt_dt_parse_iso_time
+tnt_dt_parse_iso_time_basic
+tnt_dt_parse_iso_time_extended
+tnt_dt_parse_iso_zone
+tnt_dt_parse_iso_zone_basic
+tnt_dt_parse_iso_zone_extended
+tnt_dt_parse_iso_zone_lenient
+tnt_dt_rdn
+tnt_dt_to_struct_tm
+tnt_dt_to_yd
+tnt_dt_to_ymd
+tnt_dt_to_yqd
+tnt_dt_to_ywd
+tnt_dt_weeks_in_year
```

Comment:
Thanks for the spot. I've unexported some unused functions.

```diff
@@ -454,37 +454,19 @@ title_update
 tnt_default_cert_dir_paths
 tnt_default_cert_file_paths
 tnt_dt_add_months
-tnt_dt_add_quarters
 tnt_dt_add_years
 tnt_dt_days_in_month
-tnt_dt_days_in_quarter
-tnt_dt_days_in_year
 tnt_dt_dom
 tnt_dt_dow
 tnt_dt_doy
 tnt_dt_from_rdn
-tnt_dt_from_struct_tm
-tnt_dt_from_yd
 tnt_dt_from_ymd
-tnt_dt_from_yqd
-tnt_dt_from_ywd
-tnt_dt_leap_year
 tnt_dt_month
 tnt_dt_parse_iso_date
 tnt_dt_parse_iso_time
-tnt_dt_parse_iso_time_basic
-tnt_dt_parse_iso_time_extended
-tnt_dt_parse_iso_zone
-tnt_dt_parse_iso_zone_basic
-tnt_dt_parse_iso_zone_extended
 tnt_dt_parse_iso_zone_lenient
 tnt_dt_rdn
-tnt_dt_to_struct_tm
-tnt_dt_to_yd
 tnt_dt_to_ymd
-tnt_dt_to_yqd
-tnt_dt_to_ywd
-tnt_dt_weeks_in_year
 tnt_dt_year
 tnt_iconv
 tnt_iconv_close
```

Reply:
Are you sure? I took the first dt symbol: `tnt_dt_add_months` and it is unused. Please, check again which symbols are not used and drop them. Do not hurry, it will not help to push the patch faster.

Reply:
Indeed, there are few more symbols not used in current commit, (but soon will be used in later commits)

```diff
diff --git a/extra/exports b/extra/exports
index 0b0ccd522..354ae0d4f 100644
--- a/extra/exports
+++ b/extra/exports
@@ -420,8 +420,6 @@ tnt_datetime_strftime
 tnt_datetime_to_string
 tnt_default_cert_dir_paths
 tnt_default_cert_file_paths
-tnt_dt_add_months
-tnt_dt_add_years
 tnt_dt_days_in_month
 tnt_dt_dom
 tnt_dt_dow
@@ -429,8 +427,6 @@ tnt_dt_doy
 tnt_dt_from_rdn
 tnt_dt_from_ymd
 tnt_dt_month
-tnt_dt_parse_iso_date
-tnt_dt_parse_iso_time
 tnt_dt_parse_iso_zone_lenient
 tnt_dt_to_ymd
 tnt_dt_year
```